docker-compose.dev.yml

```diff
@@ -209,48 +164,39 @@ services:
       - /dev/disk/:/dev/disk:ro
 
-
-  bibbox-sys-commander-keycloak:
+# sudo chown 1001:1000 -R dir in case we have any permission issues with keycloak.
+  bibbox-sys-commander-keycloak:         # KEYCLOAK     
     container_name: bibbox-sys-commander-keycloak
-    image: jboss/keycloak
-    networks:
-      - bibbox-default-network
+    image: bitnami/keycloak
     restart: unless-stopped
     ports:
       - "5014:8080"
+    networks:
+      - bibbox-default-network
     environment:
-      DB_VENDOR: postgres
-      DB_ADDR: bibbox-sys-commander-postgres
-      DB_DATABASE: keycloak
-      DB_USER: keycloak
-      DB_PASSWORD: keycloak
-      KEYCLOAK_IMPORT: /opt/jboss/keycloak/import/realms/realm-export.json # path to realm file in container
-      DB_WAIT_TIMEOUT: 180 # wait for postgres to initialize
-      DB_SCHEMA: public # use the public schema
-      JBOSS_CLI: "/opt/jboss/keycloak/bin/jboss-cli.sh"
-    #       PROXY_ADDRESS_FORWARDING: "true"
-    #       REDIRECT_SOCKET: "proxy-https"
-    #       KEYCLOAK_FRONTEND_URL: e.g. https://keycloak.demo.bibbox.org/auth
-
-    env_file: # we want to read some variables from the .env file as they are shared with the backend
-      - .env
-
+      - KEYCLOAK_CREATE_ADMIN_USER=true
+      - DB_ADDR=bibbox-sys-commander-postgres
+      - DB_DATABASE=keycloak
+      - DB_USER=keycloak
+      - DB_PASSWORD=keycloak
+      - DB_SCHEMA=public
+      # for changing the themes add --spi-theme-static-max-age=-1 --spi-theme-cache-themes=false --spi-theme-cache-templates=false to the extra args command below.
+      - KEYCLOAK_EXTRA_ARGS=--import-realm --health-enabled=true
     volumes:
-      - ./keycloak/realms/realm-export.json:/opt/jboss/keycloak/import/realms/realm-export.json
-      - ./keycloak/themes/custom/:/opt/jboss/keycloak/themes/custom
-      #- ./keycloak/disable-theme-cache.cli:/opt/jboss/startup-scripts/disable-theme-cache.cli
-    command: [
-      "-Dkeycloak.profile.feature.upload_scripts=enabled",
-      "-Dkeycloak.migration.action=import",
-      "-Dkeycloak.migration.provider=singleFile",
-      "-Dkeycloak.migration.file=/opt/jboss/keycloak/import/realms/realm-export.json",
-      "-Dkeycloak.migration.strategy=IGNORE_EXISTING" # this will only import the realm if it does not exist yet, if there are changes to the realm, the container needs to be deleted and recreated (this also deletes users)
-    ]
+     - ./keycloak/realms/realm-export.json:/opt/bitnami/keycloak/data/import/realm-export.json
+     - ./keycloak/themes:/opt/bitnami/keycloak/themes
+    env_file: 
+        - .env
+    healthcheck:
+       test: ["CMD-SHELL", "curl --head -fsS http://localhost:8080/health/ready"]
+       interval: 1s
+       timeout: 3s
+       retries: 60
     depends_on:
       bibbox-sys-commander-postgres:
           condition: service_healthy
 
-  bibbox-sys-commander-master-fdp:
+  bibbox-sys-commander-master-fdp:       # FDP     - no alias
```

Comment:
to be consistent either remove - no alias or it add to the others without alias'